client/Cargo.toml

```diff
@@ -32,13 +32,16 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 tungstenite = { version = "0.16" }
 smallvec = { version = "1.8.0", features = ["serde", "union"] }
-smallstr = { version = "0.2.0", features = ["serde", "union"] }
+smallstr = { git = "https://github.com/murarth/smallstr", rev = "68efc08", default-features = false, features = ["serde", "union"] }
```

Comment:
it used to be that we needed feature which was merged to `smallstr` master but wasn't included in the latest version. Now since the `0.3.0` was released we don't need to reference the specific commit anymore

Reply:
It was released 5 days ago. I'll update right now.